docs/siem/detections/api/rules-api-create.asciidoc

```diff
@@ -1,13 +1,23 @@
 [[rules-api-create]]
 === Create rule
 
-Creates a new signal detection rule.
+Creates a new detection rule.
 
-You can create two types of rules:
+You can create these types of rules:
 
-* Query-based rules, which search the defined indices and creates a signal when
+* Query rules: Searches the defined indices and creates an alert when
```

Comment:
Should we bold query and thershold?

Reply:
I'm not suggesting we bold these terms every time they are presented, just the first time in this bulleted list so readers know to look at for them throughout the rest of the doc.